### ISSUE TITLE
Select a license for your code

### ISSUE THREAD

**Issue by boostchicken** [unifi-utilities/uxg-boot#7]
I'll leave it up to you, I released everything as GPL v3 to ensure any commercial works have to copyleft.  Other than that I am a fan of MIT or CC-Zero if you just want to let people go nuts.

I am willing to discuss the wisdom of re-licensing the other repos if you decide to go a different route.

**Comment by joshuaspence** [comment]
Yeah, I had been thinking about this and wasn't sure how to handle it since the Docker container contains Ubiquiti code (NodeJS application) with no licensed information. I'm not sure whether that even matters

**Comment by boostchicken** [comment]
It does not, we are not redistributing their code directly or providing any warranties.  You can pick any license that fits you and how much control you want over the project.  Some licenses like GPLv3 transfers any patents dervied from contributions to the project owner, so that is something to consider.

**Comment by joshuaspence** [comment]
Alright, well I don't have a strong opinion on this so happy to go with GPLv3 if you say so

**Comment by boostchicken** [comment]
Its really quite irrelevant, while I have had commercial opportunities with this repository I have thought better of it.  The GPLv3 just keeps us close to that ideal.

This project will never get big enough or make some serious IP where licensing really matters.

**Comment by bslatyer** [comment]
This issue is stale because it has been open for 30 days with no activity.
